### DISCUSSION TITLE
Why minify command is not implemented in swc cli?

### DISCUSSION THREAD

**Discussion by qarmin** [swc-project/swc#9376] [General]
I see that minifier is used e.g. here - https://github.com/swc-project/swc/issues/9356, but it is not available in cli.

**Comment by kdy1** [comment]
Just because of the lack of the developer resource? It should be easy to implement, though

**Reply by qarmin** [reply]
But is the minimization logic already present on the rust side and this needs to be plugged into cli, or is there more functionality missing?

**Reply by kdy1** [reply]
Nope. Just no one worked on the CLI. That's all

**Reply by dehghani-mehdi** [reply]
At the very least, please mention this in the documentation or the CLI help text to prevent users from wasting their time.
